src/Derivative/Parser.hs

```diff
@@ -40,8 +40,7 @@ parser :: (forall v. Combinator t v a) -> Parser t a
 parser r = compact $ Graph r
 
 combinator :: Parser t a -> Combinator t v a
-combinator = unGraph
-
+combinator p = unGraph p
```

Comment:
FYI, the extra line of whitespace above heading comments is intentional in my projects. I should really make time to write that sort of thing up in a `CONTRIBUTING.md` file sometime. Regardless, I'm content with merging this as-is.